nixos/modules/system/boot/systemd-homed.nix

```
@@ -0,0 +1,55 @@
+{ config, lib, ... }:
+
+let
+  inherit (lib) mkOption mkEnableOption mkIf types;
+
+  cfg = config.systemd.homed;
+in {
+  options = {
+    systemd.homed = {
+      enable = mkEnableOption "Enable homed";
+
+      DefaultStorage = mkOption {
```

```suggestion
      defaultStorage = mkOption {
```

IMHO this would be more consistent with other nixpkgs options.

---

Made this big, because then it's the same spelling as in the file. Like systemd units also use big letters.

---

Well, NixOS module options usually start with small letters. The things starting with big letters are usually members of attrsets (and not real NixOS options) rendered by Nix.